src/travis-ci/travis-ci-create.js

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+
+const travisCIFileContents = '' +
```

Comment:
Should we make these template files like the index.js and package.json have?

Reply:
I have an issue #21 which we will do this in. The #4 has the same issue.

Reply:
Sounds good!